base/shared/src/main/scala/scalaz/data/identity.scala

```diff
@@ -1,27 +1,34 @@
 package scalaz
 package data
 
+import scala.{ List, Nil }
+
 import core.EqClass
-import ct.MonadClass
+import ct.{ BindClass, FoldableClass, MonadClass, TraversableClass }
 import debug.DebugClass
+import scalaz.algebra.MonoidClass
 
 sealed abstract class IdentityModule {
   type Identity[A]
 
   def apply[A](a: A): Identity[A]
   def run[A](id: Identity[A]): A
 
-  def identityMonad: Monad[Identity]
+  def monad: Monad[Identity]
+  def traversable: Traversable[Identity]
 
-  def identityEq[A: Eq]: Eq[Identity[A]]
-  def identityDebug[A: Debug]: Debug[Identity[A]]
+  def monoid[A: Monoid]: Monoid[Identity[A]]
+  def eq[A: Eq]: Eq[Identity[A]]
+  def debug[A: Debug]: Debug[Identity[A]]
```

Comment:
It would be cool to do this with a `Identity <~> λ[⍺ => ⍺]` or something, like we have in 7.x, but not now.

Reply:
@hrhino agreed, soon!